pkg/settings/setting.go

```diff
@@ -126,6 +126,9 @@ var (
 
 	FleetMinVersion          = NewSetting("fleet-min-version", "")
 	RancherWebhookMinVersion = NewSetting("rancher-webhook-min-version", "")
+
+	Rke2KubernetesVersion = NewSetting("rke2-default-version", "")
+	K3sKubernetesVersion  = NewSetting("k3s-default-version", "")
```

Comment:
The variable name and setting name should match.

Reply:
fixed.